exporter/sumologicexporter/README.md

```diff
@@ -99,7 +101,7 @@ exporters:
     decompose_otlp_histograms: {true, false}
 
     # timeout is the timeout for every attempt to send data to the backend,
-    # maximum connection timeout is 55s, default = 5s
+    # maximum connection timeout is 55s, default = 30s
```

Comment:
Is this maximum connection timeout ever enforced or tested? It's unrelated to this PR, so no need to do anything now, just something to consider if it's not currently enforced.

Reply:
I don't think we test it
Created issue so we can keep track of it: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33151